test/MUXControlsTestApp/InfoBar.cs

```diff
@@ -0,0 +1,473 @@
+        }
+
+        public static readonly DependencyProperty CloseButtonContentProperty =
+            DependencyProperty.Register(nameof(CloseButtonContent), typeof(object), typeof(InfoBar), new PropertyMetadata(null, OnPropertyChanged));
+
+        public Style CloseButtonStyle
+        {
+            get { return (Style)GetValue(CloseButtonStyleProperty); }
+            set { SetValue(CloseButtonStyleProperty, value); }
+        }
+
+        public static readonly DependencyProperty CloseButtonStyleProperty =
+            DependencyProperty.Register(nameof(CloseButtonStyle), typeof(Style), typeof(InfoBar), new PropertyMetadata(null));
+
+
+        public ICommand CloseButtonCommand
+        {
+            get { return (ICommand)GetValue(CloseButtonCommandProperty); }
+            set { SetValue(CloseButtonCommandProperty, value); }
+        }
+
+        public static readonly DependencyProperty CloseButtonCommandProperty =
+            DependencyProperty.Register(nameof(CloseButtonCommand), typeof(ICommand), typeof(InfoBar), new PropertyMetadata(null));
+
+
+        public object CloseButtonCommandParameter
+        {
+            get { return (object)GetValue(CloseButtonCommandParameterProperty); }
+            set { SetValue(CloseButtonCommandParameterProperty, value); }
+        }
+
+        public static readonly DependencyProperty CloseButtonCommandParameterProperty =
+            DependencyProperty.Register(nameof(CloseButtonCommandParameter), typeof(object), typeof(InfoBar), new PropertyMetadata(null));
+
+
+
+
+        /* Severity-Related Properties
+        * 
+        */
+        public InfoBarSeverity Severity
+        {
+            get { return (InfoBarSeverity)GetValue(SeverityProperty); }
+            set { SetValue(SeverityProperty, value); }
+        }
+
+        public static readonly DependencyProperty SeverityProperty =
+            DependencyProperty.Register(nameof(Severity), typeof(InfoBarSeverity), typeof(InfoBar), new PropertyMetadata(InfoBarSeverity.Default, OnPropertyChanged));
+
+        public Color StatusColor
+        {
+            get { return (Color)GetValue(StatusColorProperty); }
+            set { SetValue(StatusColorProperty, value); }
+        }
+
+        public static readonly DependencyProperty StatusColorProperty =
+            DependencyProperty.Register(nameof(StatusColor), typeof(Color), typeof(InfoBar), new PropertyMetadata(Color.FromArgb(0, 0, 0, 0)));
+
+
+        public IconSource IconSource
+        {
+            get { return (IconSource)GetValue(IconSourceProperty); }
+            set { SetValue(IconSourceProperty, value); }
+        }
+
+        public static readonly DependencyProperty IconSourceProperty =
+            DependencyProperty.Register(nameof(IconSource), typeof(IconSource), typeof(InfoBar), new PropertyMetadata(default));
+
+
+
+        // Event handlers for Close Button and Action Button
+        private void OnCloseButtonClick(object sender, RoutedEventArgs e)
+        {
```

Comment:
Whitespace throughout the file is a little inconsistent. Avoid blank lines after '{' and before '}'.